src/Compilers/VisualBasic/Portable/Symbols/ReferenceManager.vb

```diff
@@ -52,21 +52,19 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 MyBase.New(simpleAssemblyName, identityComparer, observedMetadata)
             End Sub
 
-            Protected Overrides Function GetActualBoundReferencesUsedBy(assemblySymbol As AssemblySymbol) As AssemblySymbol()
-                Dim refs As New List(Of AssemblySymbol)
+            Protected Overrides Sub GetActualBoundReferencesUsedBy(assemblySymbol As AssemblySymbol, referencedAssemblySymbols As List(Of AssemblySymbol))
+                referencedAssemblySymbols.Clear()
```

Comment:
On the VB side you clear the list, but on the C# side you assert that it's empty. Could we align the implementations?

Reply:
Agree we should align this.

Reply:
I missed this comment. Will follow up with a separate PR.